### ISSUE TITLE
wetrun testing

### ISSUE THREAD

**Issue by jordandekraker** [khanlab/hippunfold#311]
Currently, I have pulled some opensource data: https://osf.io/k2nme/ and Doc here: https://github.com/khanlab/hippunfold/pull/303

TODO:
- ideally, we should make all of these wetrun data into a snakemake workflow and automate
- ideally outputs will be kept to avoid re-running (and updated when a PR is merged) (i.e. hippunfold_outputs_version-latest on OSF) (maybe revisit with zenodo?)
- add some metrics to compare hippunfold version (or between commits)
- [ ] surface-surface differences
- [ ] subfields volume difference
- [ ] table of failed cases (e.g. >5mm difference)
- plot and view differences
- add workflow for automatically running this comparison (maybe snake-batch?)

**Comment by akhanf** [comment]
This is something @Dhananjhay is working on, should include some aspect of this (whatever is feasible) into v2.0

**Comment by Dhananjhay** [comment]
I agree! This issue on my priority list has temporarily taken a backseat to accommodate tasks pertaining to the OHBM deadline but I'll get back on it as soon as possible.
